en/services/mission.md

```diff
@@ -311,6 +311,76 @@ Note:
 The *defacto* standard file format for exchanging missions/plans is discussed in: [File Formats > Mission Plain-Text File Format](../file_formats/README.md#mission_plain_text_file).
 
 
+## Mission Command Detail
+
+This section is for clarifications and additional information about common mission items.
+In particular it is intended for cases that are difficult to document in the specification XML, or when images will much better describe expected behaviour.
+
+
+### Loiter Commands (`MAV_CMD_NAV_LOITER_*`) {#loiter_commands}
+
+Loiter commands are provided to allow a vehicle to hold at a waypoint based on time, number of turns, or indefinitely.
+Multicopter vehicles stop at the specified location (within a *vehicle-specific* acceptance radius that is not set by the mission item).
+Forward-moving vehicles (e.g. fixed-wing) circle the point with the specified radius/direction.
+
+The commands are:
+- [MAV_CMD_NAV_LOITER_TIME](../messages/common.md#MAV_CMD_NAV_LOITER_TIME) - Loiter at specified location for a given amount of time after reaching the location. 
+- [MAV_CMD_NAV_LOITER_TURNS](../messages/common.md#MAV_CMD_NAV_LOITER_TURNS) - Loiter at specified location for a given number of turns.
+- [MAV_CMD_NAV_LOITER_UNLIM](../messages/common.md#MAV_CMD_NAV_LOITER_UNLIM) - Loiter at specified location for an unlimited amount of time, yawing to face a given direction.
+
+The location and fixed-wing loiter radius parameters are common to all commands:
+
+Param (:Label) | Description | Units
+--- | --- | ---
+3: Radius | Radius around waypoint. If positive loiter clockwise, else counter-clockwise| m
+5: Latitude | Latitude | 
+6: Longitude | Longitude | 
+7: Altitude | Altitude | m
+
+The loiter time and turns are set in param 1 for the respective messages.
+The direction of loiter for `MAV_CMD_NAV_LOITER_UNLIM` can be set using `param4` (Yaw).
+
+The remaining parameters define the location at which a fixed wing vehicle will *exit the loiter, and its path to the next waypoint* (these apply only to apply to only `MAV_CMD_NAV_LOITER_TIME` and `MAV_CMD_NAV_LOITER_TURNS`). 
+
+
+Param (:Label) | Description | Units
+--- | --- | ---
+2: Heading Required	| Leave loiter circle only once heading towards the next waypoint (0 = False)| min:0 max:1 increment:1	
+4: Xtrack Location | For forward-moving aircraft this sets xtrack path or exit location: 0 for the vehicle to converge towards the center xtrack when it leaves the loiter (the line between the centers of the current and next waypoint), 1 to converge to the direct line between the location that the vehicle exits the loiter radius and the next waypoint. Otherwise the angle (in degrees) between the tangent of the loiter circle and the center xtrack at which the vehicle must leave the loiter (and converge to the center xtrack). NaN to use the current system default xtrack behaviour.
```

Comment:
> Otherwise the angle 

This refers to 'not forward moving vehicles', right? Maybe worth mentioning. But if so, if it's just for MC, then there is no loiter circle, is there? And doesn't it for MC just define the heading it has after leaves the loiter?

Reply:
The whole thing applies **only** to forward moving vehicles; xtrack does nothing at all to MC behaviour. For MC it just flies from current location towards center of next location. I will clarify that. 

The angle applies to forward flying only. This is the angle at which the vehicle leaves the loiter to head towards the center xtrack. A small angle means joining the xtrack closer to the next waypoint. A large angle (90) means joining very close to the current loiter. Many angles won't be possible if heading required is set. Basically this is something I don't think most people would use, but they could.

Reply:
okay, thanks for the clarification. The description makes sense then.